tools/archive-tool/archiveTool.cpp

```diff
@@ -154,14 +153,7 @@ TransactionalStorageInterface::Ptr createBackendStorage(
         bcos::security::DataEncryptInterface::Ptr dataEncryption = nullptr;
         if (nodeConfig->storageSecurityEnable())
         {
-            if (nodeConfig->enableHsm())
-            {
-                dataEncryption = std::make_shared<bcos::security::HsmDataEncryption>(nodeConfig);
-            }
-            else
-            {
-                dataEncryption = std::make_shared<bcos::security::DataEncryption>(nodeConfig);
-            }
+            dataEncryption = std::make_shared<bcos::security::DataEncryption>(nodeConfig);
```

Comment:
感觉这里代码可以留着，给出注释为啥不能用，后来的人再看到这里就知道有坑了，以后再来开发的时候会注意这里

```suggestion
if (nodeConfig->enableHsm())
            {
                //  Notice: [why there cannot use hsm, we need to fix xxxx for future use]
                // dataEncryption = std::make_shared<bcos::security::HsmDataEncryption>(nodeConfig);
                 dataEncryption = std::make_shared<bcos::security::DataEncryption>(nodeConfig);
            }
            else
            {
                dataEncryption = std::make_shared<bcos::security::DataEncryption>(nodeConfig);
            }
```

Reply:
has add notice for code